config/exporter/stackdriver-exporter.json

```diff
@@ -0,0 +1,10 @@
+{
+	"apiEndpoint": "staging-monitoring.sandbox.googleapis.com:443",
+	"exportPeriod": "60s",
+	"gceMetadata": {
+		"projectID": "xueweiz-experimental",
```

Comment:
Better not use personal project. What's the purpose of this config file? Is it just for demonstration? Will it be used in production?

Reply:
Removed. Sorry the personal project part is added when I was testing it. It's not useful anymore now.

The config file is intended to be used in production.
(Although an empty config file can also be used in production, because the current one is basically just explicitly specifying all the default options.)